src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -192,6 +196,17 @@ function IOURequestStepConfirmation({
     }, [transactionID, requestType, defaultCategory]);
 
     const navigateBack = useCallback(() => {
+        // If the action is categorize and there's no policies except personal one, we simply call goBack(), i.e: dismiss the whole flow together
+        if (
+            action === CONST.IOU.ACTION.CATEGORIZE &&
+            Object.values(allPolicies ?? {}).filter(
+                (individualPolicy) =>
+                    individualPolicy && individualPolicy.type !== CONST.POLICY.TYPE.PERSONAL && individualPolicy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
```

Comment:
Let's move this to PolicyUtils with something like `hasOnlyPersonalPolicy` or something.

Also, let's rename the `individualPolicy` to `policy`